src/bauhaus/bauhaus.c

```diff
@@ -2177,7 +2179,8 @@ static gboolean _widget_draw(GtkWidget *widget, cairo_t *crf)
   GdkRGBA *fg_color = default_color_assign();
   GdkRGBA *bg_color;
   GdkRGBA *text_color = default_color_assign();
-  const GtkStateFlags state = gtk_widget_get_state_flags(widget);
+  const gboolean hovering = widget == darktable.bauhaus->hovered ? GTK_STATE_FLAG_PRELIGHT : 0;
```

Comment:
Shouldn't this variable be a GtkStateFlags, since it gets or'ed with another value of that type?